rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -404,40 +400,31 @@ class DebruijnInterpreter[M[_], F[_]](
   )(env: Env[Par], rand: Blake2b512Random, sequenceNumber: Int): M[Unit] =
     eval(neu)(env, rand, sequenceNumber)
 
-  // TODO: Eliminate variable shadowing
   private def eval(
       neu: New
   )(implicit env: Env[Par], rand: Blake2b512Random, sequenceNumber: Int): M[Unit] = {
 
     def alloc(count: Int, urns: Seq[String]): M[Env[Par]] = {
-
       val simpleNews = (0 until (count - urns.size)).toList.foldLeft(env) { (_env, _) =>
         val addr: Par = GPrivate(ByteString.copyFrom(rand.next()))
         _env.put(addr)
       }
 
-      def addUrn(
-          deployParameters: DeployParameters
-      )(newEnv: Env[Par], urn: String): Either[ReduceError, Env[Par]] =
-        if (urn == "rho:deployer:auth") {
-          deployParameters.userId.exprs.headOption match {
-            case Some(Expr(GByteArray(bs))) =>
-              newEnv.put(Par(unforgeables = Vector(GDeployerAuth(bs)))).asRight[ReduceError]
-            case _ => ReduceError("No deploy parameters set").asLeft[Env[Par]]
-          }
-        } else
+      def addUrn(newEnv: Env[Par], urn: String): Either[ReduceError, Env[Par]] =
+        if (urn == "rho:deployer:auth")
```

Comment:
we should change the URI to `rho:rchain:deployerId`. It's how we refer to it all the time after all

Reply:
Done